extendr-api/src/iter.rs

```diff
@@ -145,6 +145,14 @@ impl TryFrom<&Robj> for StrIter {
     }
 }
 
+impl TryFrom<Robj> for StrIter {
```

Comment:
Can we add a doc comment here? I'm not sure where this may be used or really how.